### PR TITLE
docs: correct STACKIT IDs

### PR DESCRIPTION
**Pull request by msanft**
### Context

We got OpenStack and STACKIT IDs mixed up in our documentation previously. The OpenStack project ID is required in the `clouds.yaml` file [^1], while the STACKIT project ID is required in the Constellation config [^2]. This fixes the warning for the OpenStack project ID in `clouds.yaml`, and adds an additional warning for the STACKIT project ID in the configuration section.

[^1]: https://github.com/edgelesssys/wiki/blob/master/documentation/constellation/stackit.md#cloudsyaml
[^2]: https://github.com/edgelesssys/wiki/blob/master/documentation/constellation/stackit.md#constellation-confyaml

### Proposed change(s)

- Fix OpenStack project ID warning (said STACKIT previously).
- Add an additional warning for the STACKIT project ID.

### Checklist

- [x] Add labels (e.g., for changelog category)
- [x] Is PR title adequate for changelog?
- [x] Link to Milestone

### CODE REVIEW

**Comment by msanft** [comment]
Will wait for review / approval and then backport accordingly.

**Review by m1ghtym0 — APPROVED** [review]
Sorry, my bad, I've got things mixed up while writing the warning, so people won't mix up things 🙈 
Thanks for fixing!

File: docs/docs/getting-started/first-steps.md
Comment (m1ghtym0) [inline docs/docs/getting-started/first-steps.md:122]:
```suggestion
    `stackitProjectID` refers to the ID of your STACKIT project. The STACKIT portal also shows the OpenStack ID that's associated with your project in some places. Make sure you insert the STACKIT project ID in the `constellation-config.yaml` file. It's of the format `XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX`.
```

**Review by m1ghtym0 — CHANGES_REQUESTED** [review]

**Review by m1ghtym0 — APPROVED** [review]

**Review by burgerdev — APPROVED** [review]

**Pull request merged**
